src/lt/plugins/auto_complete.cljs

```diff
@@ -148,20 +148,34 @@
 
 (declare hinter)
 
+(defn remove-long-completitions [hints]
```

Comment:
I'm fine with, and prefer, long function names, but you probably meant to type `remove-long-completions`. "Completition" makes me think of a madlib competition.